eng/Version.Details.xml

```diff
@@ -167,11 +167,15 @@
       <Uri>https://github.com/Microsoft/ApplicationInsights-dotnet</Uri>
       <Sha>53b80940842204f78708a538628288ff5d741a1d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-8.0.100" Version="8.0.0-alpha.1.23073.2">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-8.0.100-alpha.1" Version="8.0.0-alpha.1.23077.1">
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>ff2362091c3ce19e09606a7de927670ba5d1cb81</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.net7.Manifest-8.0.100" Version="8.0.0-alpha.1.23073.2">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.net7.Manifest-8.0.100-alpha.1" Version="8.0.0-alpha.1.23077.1">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>ff2362091c3ce19e09606a7de927670ba5d1cb81</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-8.0.100-alpha.1" Version="8.0.0-alpha.1.23077.1">
```

Comment:
The `Version.Details.xml` changes look right to me. I'll need to make similar changes in xamarin-android and dotnet/maui.

/cc @rolfbjarne I don't know if you track dotnet/emsdk in xamarin-macios.